### PR TITLE
Tweet

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: source/_patterns/00-atoms/components/tweet.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  text:
+    type: array
+    items:
+      $ref: ../../00-atoms/components/paragraph.yaml
+    minItems: 1
+  date:
+    $ref: ../../00-atoms/components/date.yaml
+  hideConversation:
+    type: boolean
+    default: false
```
Comment (nlisgo) [inline source/_patterns/00-atoms/components/tweet.yaml:22]: this default should be true

File: source/_patterns/00-atoms/components/tweet.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  hideCards:
+    type: boolean
+    default: false
```
Comment (nlisgo) [inline source/_patterns/00-atoms/components/tweet.yaml:25]: this default should be true

File: source/_patterns/00-atoms/components/tweet.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "url": "https://twitter.com/eLife/status/1225868108168093696",
+  "accountId": "eLife",
+  "accountLabel": "eLife - the journal",
+  "text": [
```
Comment (nlisgo) [inline source/_patterns/00-atoms/components/tweet.json:5]: This is expected to be a single string of HTML so I will adjust it.

**Comment by discodavey** [comment]
@nlisgo The links within the actual Tweet are just code and not an actual link. Can you have a look at it please. Screen shot below
Screenshot 2020-05-18 at 15 08 08

**Comment by nlisgo** [comment]
We need a slight refactor so that the style of the tweet when javascript is enabled is unaffected.

See: http://s3.amazonaws.com/ci-pattern-library/979/patterns/00-atoms-components-tweet/00-atoms-components-tweet.html

**Comment by discodavey** [comment]
Tweet on pattern library is only showing non js version so cannot check to see if ok until gone through CI.

**Comment by nlisgo** [comment]
All representations of the tweet look good for me:

- https://s3.amazonaws.com/ci-pattern-library/979/patterns/04-pages-inside-elife-article/04-pages-inside-elife-article.html
- https://s3.amazonaws.com/ci-pattern-library/979/patterns/01-molecules-components-captioned-asset-tweet/01-molecules-components-captioned-asset-tweet.html
- https://s3.amazonaws.com/ci-pattern-library/979/patterns/00-atoms-components-tweet-with-conversation/00-atoms-components-tweet-with-conversation.html
- https://s3.amazonaws.com/ci-pattern-library/979/patterns/00-atoms-components-tweet-with-cards/00-atoms-components-tweet-with-cards.html
- https://s3.amazonaws.com/ci-pattern-library/979/patterns/00-atoms-components-tweet/00-atoms-components-tweet.html

**Comment by discodavey** [comment]
Tested on:
**PC** - Chrome and Firefox
**Mac** - Chrome, Firefox and Safari
**iPhone** - Chrome, Firefox and Safari

**Pull request merged**